cmd/kubeadm/app/constants/constants.go

```diff
@@ -64,6 +64,33 @@ const (
 	// APIServerKubeletClientCertCommonName defines kubelet client certificate common name (CN)
 	APIServerKubeletClientCertCommonName = "kube-apiserver-kubelet-client"
 
+	// EtcdCertAndKeyBaseName defines etcd's server certificate and key base name
```

Comment:
We should follow names outlined here: https://coreos.com/etcd/docs/latest/op-guide/security.html , there are a couple of conflicts.

Reply:
I'll update the etcd-server cert like @jamiehannaford suggests.

For peers, I see:
- this PR names the Peer Cert "`etcd-peer.crt`"
- the CoreOS example names it "`member1.crt`"

I find this naming strange. It doesn't match the flags: `--peer-cert-file`, `--peer-key-file`
The term "member" is used very loosely.

We have an ansible example in the contrib repo that uses the peer naming:
https://github.com/kubernetes/contrib/blob/master/ansible/roles/etcd/defaults/main.yaml#L21-L31

Reply:
This CoreOS example mentions `member.crt` in the description but uses `infra0-peer.crt` in the code:
https://github.com/coreos/etcd/blob/master/Documentation/op-guide/clustering.md#self-signed-certificates

This other CoreOS example just reuses the server crt as the peer crt:
https://github.com/coreos/etcd/blob/master/hack/tls-setup/Procfile